### PR TITLE
Update upgrade.html for 3.1.1

### PR DESCRIPTION
**Pull request by tombentley**

### CODE REVIEW

**Comment by tombentley** [comment]
@mimaison please could you review?

**Comment by mimaison** [comment]
Should we also update `quickstart.html`?

**Comment by tombentley** [comment]
Ah, good point, now done.

File: docs/upgrade.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
-<h4><a id="upgrade_3_1_0" href="#upgrade_3_1_0">Upgrading to 3.1.0 from any version 0.8.x through 3.0.x</a></h4>
+<h4><a id="upgrade_3_1_0" href="#upgrade_3_1_1">Upgrading to 3.1.1 from any version 0.8.x through 3.0.x</a></h4>
```
Comment (mimaison) [inline docs/upgrade.html]: For previous releases it looks like we also updated the `id` attribute

**Review by mimaison — APPROVED** [review]

**Pull request merged**